drivers/sensor/shtcx/shtcx.c

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
```

Comment:
`* Copyright (c) 2021 Thomas Stranger <thomas.stranger@outlook.com>` ?

Reply:
Sorry to bug you. I have seen many license headers w/o the mail.
Is it mandatory for private contributors to include the mail in each license header?